lib/zone-spec/long-stack-trace.ts

```diff
@@ -6,6 +6,8 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import {zoneSymbol} from '../common/utils';
```

Comment:
Yes, you are right, I will modify it.

By the way, I just found the same problem exists in my previous PR, for web-apis-mediaquery patch, and web-apis-notification patch, those patches are also be separated, and import function from utils. I will also modify those files  to make sure the standalone files not import from utils.